source/coffee/image-picker.coffee

```diff
@@ -179,7 +189,46 @@ class ImagePickerOption
     imgAlt = @option.data("img-alt")
     if imgAlt
       image.attr('alt', imgAlt);
-    thumbnail.on("click", @clicked)  
+    thumbnail.on("click", @clicked)
```

Comment:
This is pretty big, can you make this another method and call it here?

Reply:
Sure thing